backend/Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+pylint = "*"
+
+[packages]
+starlette = "*"
```

Comment:
Are you sure you want to use bare starlette? You could use https://github.com/tiangolo/fastapi which is a framework built upon starlette, will probably provide you with a better DX.

Reply:
I thought  it'd be an overkill to use fastapi, but it should be actually a better DX - many handy features.